pkg/monitoring/metrics/common/client/rt_wrapper.go

```diff
@@ -20,19 +20,19 @@ package client
 
 import (
 	"net/http"
+	"net/url"
 	"strconv"
 	"strings"
 )
 
+const defaultNone = "none"
+
 type rtWrapper struct {
 	origRoundTripper http.RoundTripper
 }
 
 func (r *rtWrapper) RoundTrip(request *http.Request) (response *http.Response, err error) {
-	var status string
-	var resource string
-	var verb string
-	var host string
+	var status, host string
```

Comment:
```suggestion
	var status string
	var host string
```

nit: Not sure we got a rule for that though